crates/script/src/runner.rs

```diff
@@ -50,6 +50,9 @@ impl ScriptRunner {
             }
         }
 
+        // set CREATE2 deployer from EvmOpts.
+        self.executor.set_create2_deployer(self.evm_opts.create2_deployer);
```

Comment:
there's no need to do that because this receives an executor for which it is already configured, let's remove this setter completely

Reply:
Thanks, the executor's inspector is the InspectorStack, which was set already in the InspectorStackBuilder.